mongo/datadog_checks/mongo/collectors/session_stats.py

```diff
@@ -22,8 +22,8 @@ def collect(self, api):
             sessions_count = next(
                 config_db['system.sessions'].aggregate([{"$listSessions": {"allUsers": True}}, {"$count": "total"}])
             )['total']
-        except Exception:
+        except Exception as e:
             self.log.info('Unable to fetch system.session statistics.')
```

Comment:
not related to your PR but to our log level discussion, could this start spamming the logs if the check fails? Also, not sure if it should be `info` or `error` (or debug)

Reply:
afaict this change doesn't affect the logging logic. Just as we used to catch any exception and `log.info` in response, that's what we're still doing now. We just reraise the exception after we log.

I agree that `log.info` seems like the wrong kind of level. I think at some point we gotta sit down with all these collectors and settle on a consistent error-handling strategy. Right now there's quite some diversity in what happens.

Reply:
I'll resolve this thread for now.